cmd/plume/release.go

```diff
@@ -445,3 +469,72 @@ func doAWS(ctx context.Context, client *http.Client, src *storage.Bucket, spec *
 		}
 	}
 }
+
+func doS3(spec *fcosChannelSpec) {
+	api, err := aws.New(&aws.Options{
+		CredentialsFile: awsCredentialsFile,
+		Profile:         spec.Profile,
+		Region:          spec.Region,
+	})
+	if err != nil {
+		plog.Fatalf("creating aws client: %v", err)
+	}
+
+	err = api.UpdateBucketObjectsACL(spec.Bucket, filepath.Join("prod", "streams", specChannel, "builds", specVersion), specPolicy)
```

Comment:
Maybe throw a link to https://github.com/coreos/fedora-coreos-tracker/issues/189 as a comment for now? Once we're happy with it, we can add it to the design doc or something and have a better reference link.